### PR TITLE
Confirm preemptible shows up on the GCE VM as expected.

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

File: src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  }
+
+  output {
+    String out = read_string(stdout())
+  }
+
+  runtime {
+    preemptible: "${set_preemptible}"
+    # ubuntu:trusty + curl
+    docker: "mcovarr/curly:v1"
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl]: There's really not a standard docker image with this!?

File: src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl [hunk 2]
```diff
@@ -0,0 +1,28 @@
+task check_preemption {
+  Int set_preemptible
+
+  command {
+    curl "http://metadata.google.internal/computeMetadata/v1/instance/scheduling/preemptible" -H "Metadata-Flavor: Google"
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl:5]: Very cool!!

**Review by cjllanwarne — APPROVED** [review]

**Comment by cjllanwarne** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/140/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Comment by ruchim** [comment]
Nice :+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/140/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl (hunk 1 above)
Comment (mcovarr) [reply src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl]: If there is I didn't find it.  I would much rather use something standard if that's available.

Comment (mcovarr) [reply src/main/resources/standardTestCases/confirm_preemptible/confirm_preemptible.wdl]: ooh look php has curl 😆

**Pull request merged**